docs/usage/models.md

```diff
@@ -613,6 +615,8 @@ _(This script is complete, it should run "as is")_
 This is a deliberate decision of *pydantic*, and in general it's the most useful approach. See 
 [here](https://github.com/samuelcolvin/pydantic/issues/578) for a longer discussion on the subject.
 
+Nevertheless, [strict type checking](types.md#strict-types) is supported.
```

Comment:
I would have said partially supported. We'll need to wait for https://github.com/samuelcolvin/pydantic/issues/1098 to have a full (and easy to use) strict type checking